tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -36,10 +46,11 @@ Alter the approval to publish the methodology with the release
     ...    with_release=${PUBLICATION_NAME} - Academic Year 2021/22
 
 Verify that the publication is still not visible on the public methodologies page without publishing the release
-    user navigates to methodologies page
+    user navigates to public methodologies page
     user checks page does not contain    ${PUBLICATION_NAME}
 
 Verify that the methodology is still not publicly accessible by URL without publishing the release
+    [Tags]    NotAgainstLocal
```

Comment:
This should be accessible if you're running the frontend in production mode (similar to last comment)